.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,90 @@
+name: Benchmark GTSFM on select datasets
+on: [pull_request]
+jobs:
+  door-dataset-sift:
+    name: Run GTSFM on the select dataset using SIFT front-end
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
+
+    env:
+      PYTHON_VERSION: 3.8
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache conda env
+        uses: actions/cache@v2
+        env:
+          # Increase this value to reset cache if environment_linux.yml has not changed
+          CACHE_NUMBER: 0
+        with:
+          path: ~/conda_pkgs_dir
+          key:
+            ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-${{hashFiles('environment_linux.yml') }}
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniconda-version: "latest"
+          activate-environment: gtsfm-v1
+          environment-file: environment_linux.yml
+          python-version: 3.8
+          use-only-tar-bz2: true # IMPORTANT: This needs to be set for caching to work properly!
+      - name: Environment setup
+        run: |
+          bash .github/scripts/setup.sh
+          conda info
+      - name: Download Skydio-crane-8 dataset
+        run: |
+          wget --save-cookies cookies.txt 'https://docs.google.com/uc?export=download&id=1mmM1p_NpL7-pnf3iHWeWVKpsm1pcBoD5' -O- \
+            | sed -rn 's/.*confirm=([0-9A-Za-z_]+).*/\1/p' > confirm.txt
+          wget --load-cookies cookies.txt -O skydio-8.zip \
+            'https://docs.google.com/uc?export=download&id=1mmM1p_NpL7-pnf3iHWeWVKpsm1pcBoD5&confirm='$(<confirm.txt)
```

Comment:
Very nice, thanks for adding these Ayush. Can you make the URLs stand-alone bash variables, like SKYDIO8_GDRIVE_URL=“blah”?

Reply:
Should we define a matrix and then use a for loop over the URLs?

Reply:
https://docs.github.com/en/actions/learn-github-actions/managing-complex-workflows#using-a-build-matrix

Reply:
I want the file names to be linked to the file ids. Cannot find a way to add them to a matrix.